skills/skill-alertmanager/__init__.py

```diff
@@ -18,16 +18,18 @@ async def alertmanager(self, event: Request):
                       pprint.pformat(payload))
 
         for alert in payload["alerts"]:
+            if alert["status"].upper() == "RESOLVED":
```

Comment:
This doesn't make sense for alertmanager, only for eventmanager. In alertmanager we need to report resolved alerts, why did you change this?